plugins/pcl/io/PcdWriter.cpp

```diff
@@ -79,11 +79,11 @@ Options PcdWriter::getDefaultOptions()
 }
 
 
-void PcdWriter::write(const PointBuffer& data)
+void PcdWriter::write(const PointViewPtr view)
 {
     pcl::PointCloud<XYZIRGBA>::Ptr cloud(new pcl::PointCloud<XYZIRGBA>);
     BOX3D const& buffer_bounds = data.calculateBounds();
-    pclsupport::PDALtoPCD(const_cast<PointBuffer&>(data), *cloud, buffer_bounds);
+    pclsupport::PDALtoPCD(const_cast<PointViewPtr>(view), *cloud, buffer_bounds);
```

Comment:
`Error	451	error C2440: 'const_cast' : cannot convert from 'const pdal::PointViewPtr' to 'pdal::PointViewPtr'`